roles/vmware-ci-nfs-share/tasks/main.yaml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Prepare the NFS shares
+  file:
+    path: '{{ item }}'
+    state: directory
+    owner: '{{ vmware_ci_nfs_share_owner_uid }}'
+    group: '{{ vmware_ci_nfs_share_owner_gid }}'
+  with_items:
+    - /srv/share/vms
+    - /srv/share/isos
+  become: true
+
+- name: Download ISO
```

Comment:
This is a blocker, IMO. Lets add the image to DIB to avoid download cost each job run. I can work on that

Reply:
I use the same roles outside of Zuul. So I would prefer to keep the task. This being said, I'm totally fine with the idea of pre-loading the images.